modules/ROOT/pages/dw-strings-functions-unwrap.adoc

```diff
@@ -49,14 +49,14 @@ output application/json
 [source,JSON,linenums]
 ----
 {
-  "a": null,
-  "b": null,
-  "c": "abc",
-  "d": "ABabcBA",
-  "e": "A",
-  "f": "A#",
-  "g": "#A"
-}
+   "a": null,
```

Comment:
make sure this fix is correct

Reply:
This is correct. We also had a contributor who created a PR requesting this fix.